### PR TITLE
Update deprecated code and fix oauth flows

### PR DESCRIPTION
**Pull request by mhamann**

### CODE REVIEW

File: Sources/Rownd/framework/Customizations.swift [hunk 1]
```diff
@@ -32,12 +32,12 @@ open class RowndCustomizations: Encodable {
 
     open var sheetCornerBorderRadius: CGFloat = CGFloat(25.0)
 
-    open var loadingAnimation: Lottie.Animation? = nil
+    open var loadingAnimation: LottieAnimation? = nil
```
Comment (mfmurray) [inline Sources/Rownd/framework/Customizations.swift:35]: Will this type work for cocoapods?

Comment (mhamann) [reply Sources/Rownd/framework/Customizations.swift:35]: hmm, good question. any reason why you think it might not?

**Review by BobbyRadford — APPROVED** [review]

File: Sources/Rownd/framework/Customizations.swift (hunk 1 above)
Comment (mhamann) [reply Sources/Rownd/framework/Customizations.swift:35]: good catch. just tried to run a pod build and it failed for this reason

Comment (mfmurray) [reply Sources/Rownd/framework/Customizations.swift:35]: Yeah I remember running into the same issue

Comment (mhamann) [reply Sources/Rownd/framework/Customizations.swift:35]: fixed by bumping the lottie version in podspec to 3.5.0

**Pull request merged**